python/ast/src/topython/stmt.kt

```diff
@@ -58,8 +58,19 @@ fun Return.toPython() =
 fun While.toPython() =
     "while ${test.toPython()}:\n${body.toPython().indent()}\n"
 
-fun If.toPython() =
-    "if ${test.toPython()}:\n${(if (body.isNotEmpty()) body.toPython() else "pass").indent()}\n${if (orelse.isNotEmpty()) "else:\n${orelse.toPython().indent()}\n" else ""}"  // todo: for optimization need to support elif
+fun If.toPython(): String {
+    val elsePart = when {
+        orelse.isEmpty() -> ""
+        orelse.size == 1 && orelse.single() is If -> "el${orelse.single().toPython()}"
```

Comment:
`el${orelse.single().toPython()}` -> `elif`? Just simpler :)

Reply:
`${orelse.single().toPython()}` is a full fledged `if condition:<\n>branch1<\n>else:<\n>branch2`. If we just append `el` to it, it becomes a full fledged `elif` ;)

Could you clarify what change you propose?

Reply:
Ah, sorry... Now I see it :) I thought that `${orelse.single().toPython()}` evaluates only to `if`. No change needed!

Reply:
No problem